website/docs/reference/widgets/json-form.md

```diff
@@ -32,6 +32,7 @@ These properties allow you to edit the JSON Form widget. All of these properties
 | **Animate Loading**       | When turned off, the widget will load without any skeletal animation. You can use a toggle switch to turn it on/off. You can also turn it off/on using javascript by enabling the JS label next to it.                       |
 | **Fixed Footer**          | Makes the footer sticky for long forms, so the Submit and Reset buttons are always visible on long forms.                                                                                                                    |
 | **Visible**               | Controls widget's visibility on the page. When turned off: The widget will not be visible when the app is published. It appears translucent when in Edit mode.                                                               |
+| **Hidden Fields in Data**               | When turned on, the output data will also contain data from hidden fields. The hidden field values will be reference the source data.                                                               |
```

Comment:
```suggestion
| **Hidden Fields in Data**               | When turned on, the output data is updated to contain data from hidden fields. The hidden field values are referenced from the source data.                                                               |
```

Reply:
@dilippitchika just updated it, because 'will' is being flagged by the linter. Does it look ok ? if yes we can commit and merge

Reply:
Thanks for the change Dancia, looks good to me. I apologise for the incorrect grammar here.